commands/version/command.js

```diff
@@ -40,8 +40,8 @@ exports.builder = (yargs, composed) => {
       requiresArg: true,
       defaultDescription: "origin",
     },
-    "github-release": {
-      describe: "Create an official GitHub release for every version.",
+    "release": {
```

Comment:
So this is a breaking change, as-is. I would suggest renaming the flag `create-release`, requiring a string argument from the enum `["gitlab", "github"]`, so we can be explicit about _which_ type of release to create. We then need to provide a backward-compatible hidden alias of `--github-release` to `--create-release=github`, similar to how `--skip-git` is [handled](https://github.com/lerna/lerna/blob/ea41fe91328858af0b5f5b8c715b17f1af637436/commands/version/command.js#L196-L198).

Just in case anyone has set `"githubRelease": true` in their `lerna.json`, you'll _also_ need to [deprecate the config](https://github.com/lerna/lerna/blob/ea41fe91328858af0b5f5b8c715b17f1af637436/core/project/lib/deprecate-config.js#L8)... although, argh, there's no way as-written in that file to change a boolean prop to a string val...

Here's a sketch of how I think it might look:
```patch
 module.exports = compose(
   // add new predicates HERE
+  remap("command.version.githubRelease", "command.version.createRelease", { alsoRoot: true, toValue: "github" }),
   remap("command.publish.npmTag", "command.publish.distTag", { alsoRoot: true }),
```
(`toValue` would default to the existing field value)

Reply:
Thank you, I was hoping there would be some direction on deprecation and this sounds great.

> there's no way as-written in that file to change a boolean prop to a string val...

We can take a look at that too, happy for it to be a separate PR if you would prefer.

Reply:
@evocateur How would you like to handle the `"githubRelease": false` case?

Should we make `toValue` a function `toValue = (value) => val && 'github'`?